tools/dependency/release_dates.py

```diff
@@ -51,6 +60,47 @@ def verify_and_print_latest_release(dep, repo, metadata_version, release_date):
         print(
             f'{Fore.YELLOW}*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
             f'{latest_release.tag_name}@<{latest_release.created_at}>{Style.RESET_ALL}')
+        # check for --cron flag, To run this only on github action schedule
+        # and it does not bloat CI on every push
+        if create_issue:
+            create_issues(dep, repo, metadata_version, release_date, latest_release)
+
+
+# create issue for stale dependency
+def create_issues(dep, repo, metadata_version, release_date, latest_release):
+    """Create issues in GitHub.
+
+    Args:
+        dep : name of the deps
+        repo : BaseURL of the repo
+        metadata_version :
+        release_date : old release_date
+        latest_release : latest_release (name and date )
+    """
+    access_token = os.getenv('GITHUB_TOKEN')
+    git = github.Github(access_token)
+    repo = git.get_repo('envoyproxy/envoy')
+    # Find GitHub label objects for LABELS.
+    labels = []
+    for label in repo.get_labels():
+        if label.name in LABELS:
+            labels.append(label.name)
+    if len(labels) != len(LABELS):
+        raise DependencyUpdateError('Unknown labels (expected %s, got %s)' % (LABELS, labels))
+    body = f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>:{latest_release.tag_name}@<{latest_release.created_at}>'
+    title = f'{dep} has a newer release {latest_release.tag_name}'
+    exists = repo.legacy_search_issues('open', title) or repo.legacy_search_issues('closed', title)
+    # TODO(htuch): Figure out how to do this without legacy and faster.
+    if exists:
+        print("Issue with %s already exists" % title)
+        print('  >> Issue already exists, not posting!')
+        return
+    print('Creating issues...')
+    try:
+        repo.create_issue(title, body=body, labels=LABELS)
```

Comment:
if `LABELS` is a global, no need to pass it around

Reply:
yep!!

Reply:
HI also this `repo.create_issue` is `pygithub` method so have to pass LABELS in this.

Reply:
my bad, apologies